string name
int16 amplitude